packages/web/src/scss/components/Grid/index.html

```diff
@@ -18,14 +18,14 @@ <h2 class="docs-Heading">Equal Columns</h2>
       <div class="docs-Box docs-Box--small px-0">1/12</div>
       <div class="docs-Box docs-Box--small px-0">1/12</div>
     </div>
-    <div class="Grid Grid--cols-6">
-      <div class="docs-Box docs-Box--small">1/6</div>
-      <div class="docs-Box docs-Box--small">1/6</div>
-      <div class="docs-Box docs-Box--small">1/6</div>
-      <div class="docs-Box docs-Box--small">1/6</div>
-      <div class="docs-Box docs-Box--small">1/6</div>
-      <div class="docs-Box docs-Box--small">1/6</div>
-    </div>
+    <ul class="Grid Grid--cols-6">
```

Comment:
it would be great to have this in all other technologies

Reply:
I tried to do it in React and I terribly failed with TypeScript 🤯. So for now, this is just for us to see it works at all. Extending React and Twig `Grid` with `elementType` is possible, but out of the scope of this issue.